src/styles/utils/_variables.scss

```
@@ -0,0 +1,8 @@
+$blue-color: #00acdc;
+$black-color: #000;
+$main-accent: #060b35;
+$secondary: #616070;
+$white-color: #fff;
+$cards-width: 200px;
+$cards-gap: 50px;

```

```suggestion
$cards-gap: 48px;

```
оскільки в тебе у картках `border-box`, то відстань між ними буде -2px (по 1px на кожен `border`)